server/auth/basic/auth_basic.go

```diff
@@ -167,6 +167,11 @@ func (a *authenticator) UpdateRecord(rec *auth.Rec, secret []byte) (*auth.Rec, e
 		uname = login
 	} else if err = a.checkLoginPolicy(uname); err != nil {
 		return nil, err
+	} else if uid, _, _, _, err := store.Users.GetAuthUniqueRecord(a.name, uname); err != nil {
+		return nil, err
+	} else if !uid.IsZero() {
+		// .User is changing unique name, check for duplicate unique name
```

Comment:
This comment is a little confusing.
Maybe rephrase as `The (new) user name already exists. Report an error.`?